src/app-layer-frames.h

```diff
@@ -60,7 +60,7 @@ typedef struct Frame {
 typedef struct Frames {
     uint16_t cnt;
     uint16_t dyn_size;     /**< size in elements of `dframes` */
-    uint32_t left_edge_rel;
+    uint64_t left_edge_rel;
```

Comment:
@victorjulien is this correct for frames ?

Reply:
no it's u32 intentionally, it's relative to something, I think the stream base seq. So when used it's added to that other thing.